src/components/common/Region/RegionInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import { Input as UIInput, Label as UILabel } from '@faststore/ui'
+import type { ChangeEvent, KeyboardEvent } from 'react'
+import useRegion from 'src/hooks/useRegion'
+
+const REGION_INPUT_ID = 'region-input'
+
+export default function RegionInput() {
+  const [postalCode, setPostalCodeState] = useState<string>('')
```

Comment:
it is not region, but zipCode, isn't it?

Reply:
https://en.wikipedia.org/wiki/Postal_code#:~:text=A%20postal%20code%20(also%20known,the%20purpose%20of%20sorting%20mail.
I think zip code is for some countries, but the real name is postal code.